### PR TITLE
IGNITE-22192 Prevent metric exporters double start

### PR DESCRIPTION
**Pull request by sanpwc**
https://issues.apache.org/jira/browse/IGNITE-22192

### CODE REVIEW

File: modules/metrics/src/main/java/org/apache/ignite/internal/metrics/MetricManagerImpl.java [hunk 1]
```diff
@@ -195,9 +195,15 @@ private <T extends ExporterView> void checkAndStartExporter(
         MetricExporter<T> exporter = availableExporters.get(exporterName);
 
         if (exporter != null) {
-            exporter.start(metricsProvider, exporterConfiguration);
-
-            enabledMetricExporters.put(exporter.name(), exporter);
+            enabledMetricExporters.computeIfAbsent(exporter.name(), name -> {
+                try {
+                    exporter.start(metricsProvider, exporterConfiguration);
+                    return exporter;
```
Comment (kgusakov) [inline modules/metrics/src/main/java/org/apache/ignite/internal/metrics/MetricManagerImpl.java:201]: empty line before?

File: modules/metrics/src/main/java/org/apache/ignite/internal/metrics/MetricManagerImpl.java [hunk 2]
```diff
@@ -195,9 +195,15 @@ private <T extends ExporterView> void checkAndStartExporter(
...
+                } catch (Exception e) {
+                    log.warn("Unable to start metrics exporter name=[" + exporterName + "].", e);
+                    return null;
```
Comment (kgusakov) [inline modules/metrics/src/main/java/org/apache/ignite/internal/metrics/MetricManagerImpl.java:204]: empty line before?

**Review by kgusakov — CHANGES_REQUESTED** [review]

File: modules/metrics/src/main/java/org/apache/ignite/internal/metrics/MetricManagerImpl.java (hunk 1 above)
Comment (sanpwc) [reply modules/metrics/src/main/java/org/apache/ignite/internal/metrics/MetricManagerImpl.java:202]: Why not. Fixed.

File: modules/metrics/src/main/java/org/apache/ignite/internal/metrics/MetricManagerImpl.java (hunk 2 above)
Comment (sanpwc) [reply modules/metrics/src/main/java/org/apache/ignite/internal/metrics/MetricManagerImpl.java:206]: Fixed.

**Review by kgusakov — APPROVED** [review]

**Pull request merged**